src/Controller/Api/CategoryController.php

```diff
@@ -142,14 +141,14 @@ public function postCategoryAction(Request $request)
      *  }
      * )
      *
-     * @param int     $id      A Category identifier
-     * @param Request $request A Symfony request
+     * @param int     $id      Category identifier
+     * @param Request $request Symfony request
      *
      * @throws NotFoundHttpException
      *
      * @return CategoryInterface
      */
-    public function putCategoryAction($id, Request $request)
+    public function putCategoryAction(int $id, Request $request)
```

Comment:
changing the signature is bc break

Reply:
Fixed. Thanks for catching this. I tested swagger on it.